plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ActiveMapTracker.java

```diff
@@ -57,7 +57,7 @@
     /**
      * All the maps that are currently open
      */
-    private Set<MapPart> openMaps = new CopyOnWriteArraySet<>();
+    private List<MapPart> openMaps = new CopyOnWriteArrayList<>();
```

Comment:
IMHO its should still be a set since WorkbenchParts (IEditorPart & IViewPart) are unique. Therfore MapPart Should be unique within a session. Would you aggree?